src/hooks/useTalentLayerClient.ts

```diff
@@ -0,0 +1,12 @@
+import { TalentLayerClient } from "@TalentLayer/client";
+import { useContext, useEffect, useState } from "react";
+import TalentLayerContext from "../context/talentLayer";
+
+
+const useTalentLayerClient = () => {
+    const talentLayerContext = useContext(TalentLayerContext);
```

Comment:
as most of page already used TalentLayerContext, it's even easier most of time to get the client from there